modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/ClientImpl.java

```diff
@@ -385,6 +385,34 @@
             feature);
     }
 
+    /** {@inheritDoc} */
+    @Override public void updateMetrics(UUID nodeId,
```

Comment:
Why have you moved this method up in the file? It makes it harder to review, can you please put it back? Thank you!